scripts/local/bootstrap-consumer.sh

```
@@ -276,14 +519,57 @@ else
+    # target-pointer values are operator input and must be written
+    # literally — an unquoted heredoc would shell-expand $(...) inside
+    # them (claude-code review, heredoc injection).
+    if [ ! -f "$CLAUDE_MD" ]; then
+        printf '# %s\n\nPlanning repo for the target product repository below. Coordination,\ntask specs, and reviews live here; ALL code changes happen in the\ntarget repo (see docs/CROSS-REPO-PATTERN.md in the kit).\n' \
+            "$PROJECT_NAME" > "$CLAUDE_MD"
+    fi
+
+    # Human-facing convention (KIT-ADR-0024) — agents grep for this
+    # section; seeded once, never rewritten (consumer-owned after that).
+    if ! grep -q '^## Target Repository' "$CLAUDE_MD"; then
+        printf '\n## Target Repository\n\n- **Path**: `%s`\n- **GitHub**: `%s`\n' \
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)